multicluster/service-mirror/cluster_watcher.go

```diff
@@ -192,18 +192,27 @@ func (rcsw *RemoteClusterServiceWatcher) originalResourceName(mirroredName strin
 	return strings.TrimSuffix(mirroredName, fmt.Sprintf("-%s", rcsw.link.TargetClusterName))
 }
 
-func (rcsw *RemoteClusterServiceWatcher) getMirroredServiceLabels() map[string]string {
-	return map[string]string{
-		consts.MirroredResourceLabel:  "true",
-		consts.RemoteClusterNameLabel: rcsw.link.TargetClusterName,
+func (rcsw *RemoteClusterServiceWatcher) getMirroredServiceLabels(remoteService ...*corev1.Service) map[string]string {
+	labels := make(map[string]string)
+
+	if len(remoteService) == 1 {
+		for key, value := range remoteService[0].ObjectMeta.Labels {
+			labels[key] = value
```

Comment:
We don't want to copy the label responsible for exporting a service. If we did, I think we could end up in a situation where a service mirror thinks it needs to mirror a service from a remote cluster that is already being mirrored from itself—essentially introducing a loop.

In general, I think its safest to skip any label that starts with the multicluster prefix:

```suggestion
            if strings.HasPrefix(key, consts.SvcMirrorPrefix) {
				continue
			}
            labels[key] = value
```

Reply:
Thank you! Doing this fixed the tests!